common-utils/src/test/java/edu/unc/lib/boxc/common/util/DateTimeUtilTest.java

```diff
@@ -32,65 +32,44 @@ public void after() {
     }
 
     @Test
-    public void utcToDateTimeTest() throws Exception {
-        DateTime date = DateTimeUtil.parseUTCToDateTime("2001");
-        assertEquals(2001, date.getYear());
-
-        date = DateTimeUtil.parseUTCToDateTime("2001-05-08");
-        assertEquals(2001, date.getYear());
-        assertEquals(5, date.getMonthOfYear());
-        assertEquals(8, date.getDayOfMonth());
+    public void parseUTCToDateYearTest() throws Exception {
+        var date = DateTimeUtil.parseUTCToDate("2001");
```

Comment:
So the DateTimeUtil method `parseUTCToDateTime` will go away when we have java 8 time implementation?

Reply:
yes, its only used internally and just there because we're stuck with the joda time library for the moment (DateTime is a joda time class)